src/winston-logger.ts

```diff
@@ -78,5 +78,15 @@ export const createLogger = (loggerConfiguration: LoggerConfiguration) => {
             }),
         );
     }
+
+    if (loggerConfiguration.logsFilePath) {
+        logger.add(
+            new winston.transports.File({
```

Comment:
use https://www.npmjs.com/package/winston-daily-rotate-file
so every day new file log will be created